pkg/e2e/framework/framework.go

```diff
@@ -436,7 +436,7 @@ func IsNodeReady(node *corev1.Node) bool {
 }
 
 func WaitUntilAllNodesAreReady(client runtimeclient.Client) error {
-	return wait.PollImmediate(1*time.Second, time.Minute, func() (bool, error) {
+	return wait.PollImmediate(1*time.Second, PoolNodesReadyTimeout, func() (bool, error) {
```

Comment:
Please add to commit what this timeout actually is and why we choose this value specifically.

Reply:
done